cycamore_input/predicting_the_past.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" standalone="no" ?>
+
+<simulation xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
Oops, that's a horrible mistake! I was meant to say:
 if xml inclusion using ENTITIY method is preferred, I would try it.

But so far, cyclus' xml parser works fine with xinclude without issues.